tests/test_l293d.py

```diff
@@ -23,6 +23,20 @@ def test_import(self):
         self.assertTrue(True)
         reload(d.driver)  # Revert changes
 
+    def test_create_motor_with_force_selection(self):
+        """
+        Test DC class instance creation with explicity
+        force_selection parameter
+        """
+        import l293d as d
+        print(str(d))
```

Comment:
can we rm the `print`? just wanted this in `test_import` 🙂

Reply:
yeah, I wil do that